app/pages/dashboard/contextCreator.js

```diff
@@ -1,5 +1,6 @@
 import manifest from './manifest.json';
 
-export const getContext = () => ({
+export const getContext = ({ orgId }) => ({
   ...manifest,
+  title: `${orgId} orders`,
```

Comment:
In admin we had that fancy function. Could we use that here? or even put that fancy function in BCL

Reply:
the reason I didn't want to do this for the title is because of the way it shows in the <title> tag in the page tab in the browser